pwa/src/templates/callLogDetailTemplate/CallLogDetailTemplate.tsx

```diff
@@ -36,16 +42,72 @@ export const CallLogDetailTemplate: React.FC<CallLogDetailTemplateProps> = ({ ca
-          <div>Date Created - {_getCallLog.data.dateCreated ?? "-"}</div>
-          <div>Date Modified - {_getCallLog.data.dateModified ?? "-"}</div>
+          <Table>
+            <TableBody>
+              <TableRow>
+                <TableHeader>Id</TableHeader>
+                <TableCell>{_getCallLog.data.id ?? "-"}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Source</TableHeader>
+                <TableCell>{_getCallLog.data.source.name ?? "-"}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Endpoint</TableHeader>
+                <TableCell>{_getCallLog.data.endpoint ?? "-"}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Config</TableHeader>
+                <TableCell>{"-"}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Method</TableHeader>
+                <TableCell>
+                  <div className={clsx(styles[`${_.lowerCase(_getCallLog.data.method)}Tag`])}>
+                    <Tag label={_getCallLog.data.method?.toString() ?? "no known method"} />
+                  </div>
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Response Status</TableHeader>
+                <TableCell>{_getCallLog.data.responseStatus || "-"}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Response Status Code</TableHeader>
+                <TableCell>
+                  <div
+                    className={clsx(
+                      styles[getStatusColor(_getCallLog.data.responseStatusCode.toString() ?? "no known status")],
+                    )}
+                  >
+                    <Tag
+                      icon={
+                        <FontAwesomeIcon
+                          icon={getStatusIcon(_getCallLog.data.responseStatusCode.toString() ?? "no known status")}
+                        />
+                      }
+                      label={_getCallLog.data.responseStatusCode?.toString() ?? "no known status"}
+                    />
+                  </div>
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Date Created</TableHeader>
+                <TableCell>{dateTime(_getCallLog.data.dateCreated) ?? "-"}</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Date Modified</TableHeader>
+                <TableCell>{dateTime(_getCallLog.data.dateModified) ?? "-"}</TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
+
+          {_getCallLog.data.responseBody && (
+            <>
+              <Heading3>Response body</Heading3>
+              <iframe className={styles.respondeBody} sandbox="" srcDoc={_getCallLog.data.responseBody}></iframe>
+            </>
+          )}
```

Comment:
I think we should always display the Header with Response body and if the log has a body we show the body and if the log doesn't have a body we should display `This callLog has no response body.`